tutorials/basics/2_basics.py

```diff
@@ -133,17 +129,17 @@
 
 
 ###############################################################################
-# One can also remove node/edge states from the graph. This is particularly
+# You can also remove node or edge states from the graph. This is particularly
 # useful to save memory during inference.
 
 g.ndata.pop('x')
 g.edata.pop('w')
 
 
 ###############################################################################
-# Multigraphs
+# Working with multigraphs
 # ~~~~~~~~~~~
-# Many graph applications need multi-edges. To enable this, construct :class:`DGLGraph`
+# Many graph applications need multiple edges. To enable this, construct :class:`DGLGraph`

```

Comment:
Maybe use "parallel edges" to avoid potential ambiguity?

Reply:
```suggestion
# Many graph applications need parallel edges. To enable this, construct :class:`DGLGraph`
```